resources/web/wwi/ProtoManager.js

```diff
@@ -59,7 +59,27 @@ export default class ProtoManager {
   async loadX3d() {
     let xml = this.getXmlOfMinimalScene();
     const scene = xml.getElementsByTagName('Scene')[0];
-    scene.appendChild(this.proto.toX3d());
+    if (this.proto.isRoot && (this.proto.baseType.name === 'PBRAppearance' || this.proto.baseType.name === 'Appearance')) {
```

Comment:
should we perhaps isolate this into a function the same way  we do for the minimal scene? I believe we said at some point that there would be other exceptions such as these (only hingejointwithbacklash comes to mind atm)

Reply:
good idea